.flowconfig

```diff
@@ -0,0 +1,51 @@
+[ignore]
```

Comment:
This file is necessary to make Flow work with the `react-native` `devDependency`, and it was created by running `react-native-git-upgrade`. I modified a couple lines since we don't actually have a compilation target here.